src/styles/index.scss

```diff
@@ -1,5 +1,8 @@
+$theme-backgroundColor: #f5f5f5;
+
 body {
     margin: 0 auto;
+    background-color: $theme-backgroundColor;
```

Comment:
can you remove the bg color it doesn't look good 
![image](https://github.com/Real-Dev-Squad/website-status/assets/70854507/c35ce981-00fe-45f2-b472-8d45d7b7fee9)